source/deployment/sso-saml-okta.rst

```diff
@@ -16,60 +16,65 @@ Set Up a Connection App for Mattermost SSO
+ - **Audience URI:** For instance, ``mattermost``
+ - **Name ID format:** ``unspecified``
+ - **Application username:** ``Email``
 
 	.. image:: ../../source/images/okta_3_initial_saml_settings.PNG
 
-6. Set up encryption for your SAML connection. First, click **Show Advanced Settings**.
+6.  To set up encryption for your SAML connection select **Show Advanced Settings**.
 
 	.. image:: ../../source/images/okta_4_initial_saml_settings.PNG
 
-Then, set **Assertion Encryption** as ``Encrypted`` and upload the **Service Provider Public Certificate** you generated in step 2 to the **Encryption Certificate** field.
+7. Set **Assertion Encryption** as **Encrypted** and upload the Service Provider Public Certificate you generated in step 2 to the **Encryption Certificate** field.
 
 	.. image:: ../../source/images/okta_5_advanced_saml_settings.PNG
 
-7. Enter attribute statements, which will be used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`. Email and username attributes are required. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
+8. Enter attribute statements, which will be used to map attributes between Okta and Mattermost. For more information on which attributes are configurable, see our :ref:`documentation on SAML configuration settings <saml-enterprise>`. Email and username attributes are required. For Mattermost servers running 3.3 and earlier, first name and last name attributes are also required.
 
 	.. image:: ../../source/images/okta_6_attribute_statements.PNG
 
-8. Click **Next**. Then, set Okta support parameters for the application. Recommended settings:
- - I’m an Okta customer adding an internal app
- - This is an internal app that we have created
+9. Click **Next**. Then, set Okta support parameters for the application. Recommended settings:
+ - **I’m an Okta customer adding an internal app**
+ - **This is an internal app that we have created**
 
 	.. image:: ../../source/images/okta_7_support_configuration.PNG
 
-9. Click **Finish**. On the next screen, click the **Sign On** tab and click **View Setup Instructions**.
+10. Click **Finish**. On the next screen, click the **Sign On** tab and click **View Setup Instructions**.
+  10.1 Select the **Identity Provider metadata** link and copy the link from the browser URL box. This will be used during the SAML configuration steps in the next section. 
 
 	.. image:: ../../source/images/okta_8_view_instructions.PNG
 
-10. Take note of ``Identity Provider Single Sign-On URL`` (also known as ``SAML SSO URL``), and the Identity Provider Issuer, as both will be needed to configure SAML for Mattermost.
+11. Take note of **Identity Provider Single Sign-On URL** (also known as **SAML SSO URL**), and the Identity Provider Issuer, as both will be needed to configure SAML for Mattermost.
```

Comment:
unless they take the alternate metadataUrl route - maybe replace "will be" with "might be"?

Reply:
Thanks Catalin, will do that.

Reply:
Forgot to mention - same observation for "Identity Provider Public Certificate"

Reply:
Thanks, done!